pkg/ddc/jindofsx/transform.go

```diff
@@ -140,7 +140,9 @@ func (e *JindoFSxEngine) transform(runtime *datav1alpha1.JindoRuntime) (value *J
 	e.transformPlacementMode(dataset, value)
 	e.transformRunAsUser(runtime, value)
 	e.transformTolerations(dataset, runtime, value)
-	e.transformResources(runtime, userQuotas, value)
+	e.transformMasterResources(runtime, value, userQuotas)
```

Comment:
I suggest keeping function `transformResources` and there are three steps in the function: `transformMasterResources`, `transformWorkerResources`, `transformFuseResources`